static/app/views/automations/components/editConnectedMonitors.tsx

```diff
@@ -35,7 +36,16 @@ export default function EditConnectedMonitors({connectedIds, setConnectedIds}: P
         {connectedMonitors.length > 0 ? t('Other Monitors') : t('All Monitors')}
       </Heading>
       <div style={{flexGrow: 1}}>
-        <StyledSearchBar placeholder={t('Search for a monitor or project')} />
+        <StyledInputGroup>
+          <InputGroup.LeadingItems disablePointerEvents>
+            <IconSearch color="subText" size="sm" />
+          </InputGroup.LeadingItems>
+          <InputGroup.Input
+            placeholder={t('Search for a monitor or project')}
+            type="text"
+            autoComplete="off"
+          />
+        </StyledInputGroup>
```

Comment:
the `SearchBar` was throwing errors about nesting a `Form` inside a `Form`, this is kind of a temporary fix since this search bar doesn't work right now anyways